### PR TITLE
Mule 16617: Email as attachment is not recognised by the Email connector

### PR DESCRIPTION
**Pull request by SebaElizalde**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -297,7 +297,7 @@
 
         <formatterConfigPath>formatter.xml</formatterConfigPath>
         <formatterGoal>validate</formatterGoal>
-        <oldMuleArtifactVersion>4.2.0-SNAPSHOT</oldMuleArtifactVersion>
+        <oldMuleArtifactVersion>4.2.1-SNAPSHOT</oldMuleArtifactVersion>
```
Comment (ndinu) [inline pom.xml]: Is this still necessary? This was probably already fixed.

Comment (SebaElizalde) [reply pom.xml]: Indeed.

**Comment by SebaElizalde** [comment]
--validate

**Comment by SebaElizalde** [comment]
--validate

**Comment by SebaElizalde** [comment]
--validate

**Comment by SebaElizalde** [comment]
--validate

**Pull request merged**